pipeline-module/{{cookiecutter.handle|lower|slugify}}/README.md

```diff
@@ -1,7 +1,7 @@
 # {{ cookiecutter.handle }}: {{ cookiecutter.module_name }}
 
-![Pipeline Level: {{ cookiecutter.level }}](https://img.shields.io/badge/Pipeline-{{ cookiecutter.level }}-blue)
-![Diagram Index: {{ cookiecutter.diagram_index }}](https://img.shields.io/badge/Diagram__Index-{{ cookiecutter.diagram_index }}-blue)
+![Pipeline Level: {{ cookiecutter.level }}](https://img.shields.io/badge/Pipeline-{{ cookiecutter.level }}-{% if cookiecutter.level == "L1" %}88BFFE{% elif cookiecutter.level == "L2" %}1AC604{% elif cookiecutter.level == "L3" %}BEB1D0{% else %}FEBCFF{% endif %})
```

Comment:
This is a bit frivolous, but cute.  We have a standard PDF showing all the pipeline modules - it's the diagram referred to by "diagram index" and I thought I'd just color-code the level badges to match the colors used on that diagram, which I just picked up from inspecting the PDF.